kornia/augmentation/container/augment.py

```diff
@@ -220,7 +231,11 @@ def inverse(  # type: ignore
                     raise ValueError(f"Unsupported Sequential {module}.")
                 else:
                     raise NotImplementedError(f"data_key {dcate} is not implemented for {module}.")
-            outputs.append(input)
+            if isinstance(arg, (Boxes,)):
+                arg._data = input
```

Comment:
I think we should have a direct method `Boxes.to_tensor()`

Reply:
or expose `data` -> https://deepsource.io/gh/kornia/kornia/run/c16adc3c-beee-4661-8387-49948b333146/python/PYL-W0212/

Reply:
This is for assigning. There is `to_tensor` and `data` already. But I meant to update the inner property than reconstructing everytime.

Reply:
then we need a setter